### PR TITLE
Feat: Add reset history

### PR DESCRIPTION
**Pull request by Biorn1950**
- add the new reset history endpoint
- fix update job instead of clear

Signed-off-by: Biorn1950 biorn@me.com

### CODE REVIEW

**Comment by Biorn1950** [comment]
I have to remove bad definition for `history.msg.confirm` and i18n Ally refracted at the same time. Which is not a bad thing IMHO.

File: src/components/widgets/stats/PrinterStatsCard.vue [hunk 1]
```diff
@@ -105,6 +118,18 @@ export default class PrinterStatsCard extends Vue {
   get supportsHistoryComponent () {
     return this.$store.getters['server/componentSupport']('history')
   }
+
+  handleResetStats () {
+    this.$confirm(
+      this.$tc('app.history.msg.confirm_stats'),
+      { title: this.$tc('app.general.label.confirm'), color: 'card-heading', icon: '$error' }
+    )
+      .then(res => {
```
Comment (pedrolamas) [inline src/components/widgets/stats/PrinterStatsCard.vue:127]: My personal preference goes to use `async..await` when possible (it should work fine here) but I have no problem with promise-style code!

File: src/api/socketActions.ts [hunk 2]
```diff
@@ -346,19 +346,25 @@ export const SocketActions = {
...
+        dispatch: 'history/onDelete',
         params
       }
     )
   },
 
+  async serverHistoryResetTotals () {
+    baseEmit(
+      'server.history.reset_totals', {
+        dispatch: 'history/onHistoryChange'
```
Comment (pedrolamas) [inline src/api/socketActions.ts:363]: This has no payload, so what will this actually do?

File: src/components/widgets/stats/PrinterStatsCard.vue (hunk 1 above)
Comment (pedrolamas) [reply src/components/widgets/stats/PrinterStatsCard.vue:127]: Fair enough! 🙂

**Review by pedrolamas — APPROVED** [review]
LGTM!

**Pull request merged**